diff --git a/libgdf/src/tests/CMakeLists.txt b/libgdf/src/tests/CMakeLists.txt
index abf3d2f..430e918 100644
--- a/libgdf/src/tests/CMakeLists.txt
+++ b/libgdf/src/tests/CMakeLists.txt
@@ -42,19 +42,19 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR};
                     ${CMAKE_CURRENT_SOURCE_DIR}/..)
 
 add_subdirectory(foo-sample)
-add_subdirectory(datetime)
-add_subdirectory(hashing)
-add_subdirectory(join)
-add_subdirectory(sqls)
-add_subdirectory(hash_map)
-add_subdirectory(groupby)
-add_subdirectory(unaryops)
-add_subdirectory(filterops_numeric)
-add_subdirectory(quantiles)
-add_subdirectory(column)
-add_subdirectory(validops)
-add_subdirectory(memory)
-add_subdirectory(gdf)
-add_subdirectory(csv)
+#add_subdirectory(datetime)
+#add_subdirectory(hashing)
+#add_subdirectory(join)
+#add_subdirectory(sqls)
+#add_subdirectory(hash_map)
+#add_subdirectory(groupby)
+#add_subdirectory(unaryops)
+#add_subdirectory(filterops_numeric)
+#add_subdirectory(quantiles)
+#add_subdirectory(column)
+#add_subdirectory(validops)
+#add_subdirectory(memory)
+#add_subdirectory(gdf)
+#add_subdirectory(csv)
 
 message(STATUS "******** Tests are ready ********")
